webpack.js

```diff
@@ -27,7 +29,10 @@ const config = {
 		],
 	},
 	plugins: [
-		new webpack.DefinePlugin({ isTesting }),
+		new webpack.DefinePlugin({
+			isTesting,
+			PLYR_ICONS: JSON.stringify(fs.readFileSync(path.join('img', 'plyr.svg'), { encoding: 'utf8' })),
```

Comment:
Thanks, I was also trying to figure things out :)

Somehow I am nor fond of loading files with node here, but I figure this is okay.
Could you put the file read after the require on the top, and maybe we can directly use the `node_modules/plyr/dist/plyr.svg` version and avoid shipping a clone of our own? (might lead to outdated data)

Thanks a lots for this, great work! :tada:

Reply:
I like that idea, sure!